slice/IceGrid/Descriptor.ice

```diff
@@ -42,6 +42,7 @@ module IceGrid
         Ice::StringSeq references;
 
         /// The property set properties.
+        ["matlab:identifier:propertyDescriptors"]
         PropertyDescriptorSeq properties;
```

Comment:
`properties` is a reserved identifier in MATLAB.

Reply:
It's not a MATLAB keyword. In particular:
```

>> matlab.lang.makeValidName('properties')

ans =

    'properties'
```

Reply:
I guess there must be some level of context-awareness here?
The docs say that it is a `"MATLAB class-definition keyword"`:
https://www.mathworks.com/help/matlab/ref/properties.html#d126e1399987

Even if it's theoretically fine, I'd still rather play it safe and remap it to something else here.
Since semantics aside, seeing `properties` in MATLAB has an understood meaning
that is different than what this sequence represents.